components/collector/tests/source_collectors/cloc/test_loc.py

```diff
@@ -56,4 +67,6 @@ async def test_loc_by_file_include_files(self):
         """Test that the number of lines is returned and that the languages are returned as entities."""
         self.set_source_parameter("files_to_include", ["test.*"])
         response = await self.collect(get_request_json_return_value=self.cloc_by_file_json)
-        self.assert_measurement(response, value="60", total="90", entities=self.expected_entities[:1])
+        expected_entities = self.expected_entities[:1]
+        expected_entities[0]["code_percentage"] = "100"
+        self.assert_measurement(response, value="60", total="90", entities=expected_entities)
```

Comment:
Filtered languages are removed from the total, filtered files are not. This is so that users can calculated the percentage of test code compared to the total number of loc.